src/store/actions/index.ts

```diff
@@ -15,10 +15,11 @@ export function toggleInputDisabled(): actionsTypes.ToggleInputDisabled {
   };
 }
 
-export function addUserMessage(text: string, id?: string): actionsTypes.AddUserMessage {
+export function addUserMessage(text: string, showClientAvatar: boolean, id?: string): actionsTypes.AddUserMessage {
```

Comment:
I think this should be only a prop, why add it to the function that creates a new message?